source/retrieve-many-applications.html.md.erb

```diff
@@ -12,9 +12,13 @@ GET /applications
 ## Example request
 
 ```
-GET /applications/:id
+GET /applications/:id?since=2018-10-01
```

Comment:
Might also be worth including a time here- Several providers use an overnight batch job to pull in new applications at defined times (6AM, 12PM) over the ODBC and XML links.